.github/workflows/ci.yml

```diff
@@ -24,12 +24,58 @@ jobs:
         with:
           python-version: ${{ matrix.python_version }}
 
-      # Use our tasks in the playbook which is about as fast.
-      # - name: Set up KinD cluster
-      #   uses: engineerd/setup-kind@v0.3.0
-
       - name: Install ansible and other dependencies.
         run: pip install ansible ansible-lint yamllint docker molecule openshift
 
       - name: Run molecule default test scenario
         run: molecule test
+
+  ansible-acd:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python_version: ['3.7']
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v1
+
+      - name: Set up Python ${{ matrix.python_version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python_version }}
+
+      - name: Install Ansible ACD distribution.
+        run: >-
+          pip install ansible>=2.10
+          --extra-index-url https://toshio.fedorapeople.org/ansible/acd/
+
+      - name: Install other dependencies.
```

Comment:
@geerlingguy this and the previous steps have trailing dots, while others don't. Maybe improve consistency?

Reply:
Oops! Yeah, I typically always end in a full stop, but some people don't, and so I sometimes confuse myself.